WordPress/Classes/ViewRelated/Site Creation/Web Address/DesignSystem/Gallery/IconGallery.swift

```diff
@@ -0,0 +1,49 @@
+import SwiftUI
+
+struct IconGallery: View {
```

Comment:
What is the state of Icons and Icons Gallery in the scope of this PR?

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/9cba5114-24bd-48c8-aede-fd330e555ce0" alt="image" width="200">

If we're making Icons as part of the Design System, should we define them in a similar way as components, and styles, rather than within a gallery?

Reply:
This is totally WIP. I should've mentioned that. I am not sure how to proceed here yet. If it looks confusing we can remove it from the PR.